pkg/gardenlet/controller/managedseed/reconciler.go

```diff
@@ -52,6 +52,9 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
 	log := logf.FromContext(ctx)
 
+	ctx, cancel := context.WithTimeout(ctx, r.Config.Controllers.ManagedSeed.SyncPeriod.Duration)
```

Comment:
Here `SyncPeriod` has defaulted to 1h. Is this the right value to use for context timeout?